features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/package-lock.json

```diff
@@ -7,7 +7,7 @@
     "acorn": {
       "version": "5.3.0",
       "resolved": "https://registry.npmjs.org/acorn/-/acorn-5.3.0.tgz",
-      "integrity": "sha512-Yej+zOJ1Dm/IMZzzj78OntP/r3zHEaKcyNoU2lAaxPtrseM6rF0xwqoz5Q5ysAiED9hTjI2hgtvLXitlCN1/Ug==",
+      "integrity": "sha1-dEbTlFnFT7SagObuZHgUm5QOyCI=",
```

Comment:
This change of integrity algo has been caused by an known issue in npm [[1](https://github.com/npm/npm/issues/16938#issuecomment-350397663)] , and it has been fixed in [npm 5.6](https://github.com/npm/npm/releases/tag/v5.6.0) need to update the npm version to the latest